admin-dev/themes/new-theme/scss/components/_tinymce.sass

```diff
@@ -548,14 +548,14 @@ body .mce-abs-layout-item
             border: 0
             padding: 0
 
-.mce-primary
+.mce-primary.mce-btn
     min-width: 50px
     color: #fff
     border: 1px solid #b1b1b1
     border-color: rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.25) rgba(0, 0, 0, 0.25)
     background-color: #006dcc
     &:hover, &:focus
-        background-color: #005fb3
+        background-color: #005fb3 !important
```

Comment:
Hmm, I totally dislike the `!important`, it prevents people to update the value with another css file.

In my opinion the change to apply must be done elsewhere.

Reply:
Hello @Quetzacoalt91 
I had already tried to modify `js/tiny_mce/skins/prestashop/skin.min.css` by removing line as you have suggested. But when I go to back-office, removed css style still there after clearing cache also. So I thinked that it's loads by default with js  :) and then tried to modify `admin-dev/themes/new-theme/scss/components/_tinymce.sass`.

I also dislike to use !important but here I need to override the style `.mce-btn:focus, .mce-btn:hover { 
     background-color: transparent !important; 
 } ` from `js/tiny_mce/skins/prestashop/skin.min.css` file. That's why I have used style with !imortant.

Reply:
I managed to make it work on my side, did you make sure your browser cache was emptied?

Reply:
@Quetzacoalt91 I have cleared browser cache and it's fixed for me now.

Reply:
@Quetzacoalt91 Thanks. I would like to thank you @matks also as you helped me a lot on Gitter regarding this PR 😀

Reply:
And now you can remove the changes applied in this file and `admin-dev/themes/new-theme/public/theme.css`, as they are properly applied on the popup correctly. :)

Reply:
@Quetzacoalt91  Does I need to remove old commits as I'm not good to remove it :).